contents/data_vis_makie_create_figure.md

```diff
@@ -0,0 +1,100 @@
+## Create Plot Figure {#sec:datavisMakie_create_Figure}
+The basic container object in Makie is `Figure`, a canvas where we can add objects like `Axis`, `Colorbar`s, `Legend`s, etc.
+
+For `Figure`, we have have attributes like `backgroundcolor`, `resolution`, `font` and `fontsize` as well as the `figure_padding` which changes the amount of space around the figure content, see the colored area in @fig:fig_axis.
+It can take one number for all sides, or a tuple of four numbers for left, right, bottom and top.
+
+```jl
+@sco JDS.figure_canvas()
+```
+
+`Axis` has a lot more, some of them are  `backgroundcolor`, `xgridcolor` and `title`.
+For a full list just type `help(Axis)`.
+
+```jl
+@sco JDS.figure_axis()
+```
+
+Now, we add a plotting function into our new Axis:
+
+```jl
+@sco JDS.figure_axis_plot()
+```
+
+This example has already most of the attributes that most users will normally use.
+Probably, a `legend` will also be good to have.
+Which for more than one function will make more sense.
```

Comment:
```suggestion
This example already includes many of the attributes that are typically used.
Additionally, it would be beneficial to include a "legend" for reference, especially if the example has more than one function.
This will make it easier to understand.
```
Thanks to ChatGPT with the query: "Make the following text flow more: [...]"